src/ui/SearchInterface/FacetValueStateHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { QueryStateModel } from '../../models/QueryStateModel';
+import { Component } from '../Base/Component';
+import { BaseComponent } from '../Base/BaseComponent';
+
+type QueryState = { [key: string]: any };
```

Comment:
It's not a super helpful type, as in it does not help the compiler find error in a meaningful way. Why not simply use `Record<any>` ?

Reply:
I agree it is _easier_ to read but it kind of hides the fact that `queryState` has a loose type. Using the actual type directly in the constructor would better expose what kind of object we're dealing with. 

It also would avoid confusing this concept with the queryStateModel. All in all, I just think that readability here is not enough of an argument.